source/tutorials/other/deploying-nextcloud.rst

```diff
@@ -2,19 +2,17 @@
 Deploying Nextcloud on an instance
 ##################################
 
+This tutorial assumes that you have the following prepared:
 
+* An SSH key.
 
-This tutorial assumes the following:
+* A domain name.
 
-* Have an SSH key.
+* A record pointing to localhost (127.0.0.1) under a chosen host name.
```

Comment:
Record is not necessary, ability to configure domain name entries programmatically. (e.g. create DDNS entry if using namecheap.com for dns)